### PR TITLE
Added python unit test homework - Atanas

### PR DESCRIPTION
**Pull request by atanasatanasov03**
Reviewerite da vnimavat >:D

### CODE REVIEW

File: homework_03/03_atanas_atanasov/circles.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class circleStates(Enum):
+	NO_COMMON_POINTS = 0
+	TOUCHING = 1
+	INTERSECTING = 2
+	SAME = 3		
+
+class circle:
+	center = 0
+	radius = 0
+
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/circles.py:23]: There is no need to define the members at the start of the class

File: homework_03/03_atanas_atanasov/circles.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+		self.center = c
+		self.radius = r
+
+	def toString(self):
+		print("center coordinates:")
+		self.center.toString()
+		print("radius: ", self.radius)
+
+	def find_relative_position(self, circle):
+		if(self.center.xcoord == circle.center.xcoord and self.center.ycoord == circle.center.ycoord):
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/circles.py:34]: You should add a check if radiuses are equal.

File: homework_03/03_atanas_atanasov/test_circles.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+	c1 = circle(p1, 1.5)
+	c2 = circle(p2, 1.25)
+	c3 = circle(p1, 2)
+	c4 = circle(p2, 3)
+
+	assert c1.find_relative_position(c3) == circleStates.SAME
+	assert c2.find_relative_position(c4) == circleStates.SAME
+
+	print("Test successful")
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/test_circles.py:67]: Tests are ok.

File: homework_03/03_atanas_atanasov/circles.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		
+		distSq = (self.center.xcoord - circle.center.xcoord) ** 2 + (self.center.ycoord - circle.center.ycoord) ** 2
+		radSumSq = (self.radius + circle.radius) ** 2;
+		
+		if (distSq == radSumSq):
+			return circleStates.TOUCHING
+		elif (distSq > radSumSq):
+			return circleStates.NO_COMMON_POINTS
+		elif (distSq < radSumSq):
+			return circleStates.INTERSECTING
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/circles.py:45]: I think checks are ok.

File: homework_03/03_atanas_atanasov/circles.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+		self.center.toString()
+		print("radius: ", self.radius)
+
+	def find_relative_position(self, circle):
+		if(self.center.xcoord == circle.center.xcoord and self.center.ycoord == circle.center.ycoord):
+			return circleStates.SAME
+		
+		distSq = (self.center.xcoord - circle.center.xcoord) ** 2 + (self.center.ycoord - circle.center.ycoord) ** 2
+		radSumSq = (self.radius + circle.radius) ** 2;
+
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/circles.py:39]: Great idea that you have added them as variables.

File: homework_03/03_atanas_atanasov/test_circles.py [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+print("Testing find_relative_position() function\n")
+test_inside()
+test_touching()
+test_noCommonPoints()
+test_same()
```
Comment (Vladikolev0321) [inline homework_03/03_atanas_atanasov/test_circles.py:73]: You can run test using pytest library instead of calling the functions.

**Review by Vladikolev0321 — APPROVED** [review]
You have done the task well, you can change the things i had mentioned.

File: homework_03/03_atanas_atanasov/circles.py (hunk 1 above)
Comment (Alexander1022) [reply homework_03/03_atanas_atanasov/circles.py:23]: Vladi is right, you don't have to define it at the start of the class.

File: homework_03/03_atanas_atanasov/circles.py (hunk 4 above)
Comment (Alexander1022) [inline homework_03/03_atanas_atanasov/circles.py:45]: The whole file looks good to me, despite the fact you doing one mistake making classes.

**Review by Alexander1022 — APPROVED** [review]
Nasko has done the homework fine and without mistakes. I have some remarks, but the code and the tests are good enough. Go get some rest!

**Pull request merged**